README.md

```diff
@@ -20,53 +20,35 @@ Contributing to FoundationDB can be in contributions to the code base, sharing y
 
 ### Binary downloads
 
-Developers interested in using the FoundationDB store for an application can get started easily by downloading and installing a binary package. Please see the [downloads page](https://www.foundationdb.org/download/) for a list of available packages.
+Developers interested in using FoundationDB can get started by downloading and installing a binary package. Please see the [downloads page](https://www.foundationdb.org/download/) for a list of available packages.
 
 
 ### Compiling from source
 
 Developers on an OS for which there is no binary package, or who would like
 to start hacking on the code, can get started by compiling from source.
 
-Currently there are two build systems: a collection of Makefiles and a
-CMake-based build system. Both of them should currently work for most users,
-and CMake should be the preferred choice as it will eventually become the only
-build system available.
+The official docker image for building is `foundationdb/foundationdb-build`. It has all dependencies installed. Use `-DBOOST_ROOT=/opt/boost_1_72_0/` in your cmake command. To build outside the official docker image you'll need at least these dependencies:
+
+1. Install cmake Version 3.13 or higher [CMake](https://cmake.org/)
+1. Download version 1.72 of [Boost](https://dl.bintray.com/boostorg/release/1.72.0/source/boost_1_72_0.tar.bz2)
+1. Unpack boost (you don't need to compile it)
+1. Install [Mono](http://www.mono-project.com/download/stable/)
+1. Install [Ninja](https://ninja-build.org/) (optional, but recommended)
 
 If compiling for local development, please set `-DUSE_WERROR=ON` in
 cmake. Our CI compiles with `-Werror` on, so this way you'll find out about
 compiler warnings that break the build earlier.
 
-## CMake
-
-To build with CMake, generally the following is required (works on Linux and
-Mac OS - for Windows see below):
+Once you have your dependencies, you can run cmake and then build:
 
 1. Check out this repository.
-1. Install cmake Version 3.13 or higher [CMake](https://cmake.org/)
-1. Download version 1.67 of [Boost](https://sourceforge.net/projects/boost/files/boost/1.67.0/). 
-1. Unpack boost (you don't need to compile it)
-1. Install [Mono](http://www.mono-project.com/download/stable/).
-1. Install a [JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html). FoundationDB currently builds with Java 8.
 1. Create a build directory (you can have the build directory anywhere you
-   like): `mkdir build`
-1. `cd build`
-1. `cmake -GNinja -DBOOST_ROOT=<PATH_TO_BOOST> <PATH_TO_FOUNDATIONDB_DIRECTORY>`
+   like). There is currently a directory in the source tree called build, but you should not use it. See [#3098](https://github.com/apple/foundationdb/issues/3098)
+1. `cd <PATH_TO_BUILD_DIRECTORY>`
+1. `cmake -G Ninja -DBOOST_ROOT=<PATH_TO_BOOST> <PATH_TO_FOUNDATIONDB_DIRECTORY>`
```

Comment:
I don't think the `-DBOOST_ROOT` is needed here. If boost is installed on the system cmake will find it. Without going into details I would mention the following: If cmake can't find boost, `BOOST_ROOT` can be set (with a link to the cmake documentation). By default cmake will download and unpack boost for you.

Reply:
Sounds good. Let's keep it simple and just let cmake do its thing. FWIW I think cmake fails to find boost sometimes, for example in the pull request builds it seems not to find it.